### PR TITLE
CLI: Add Docker commands

### PR DESCRIPTION
**Pull request by samiff**
#### Changes proposed in this Pull Request:

* Straight pass through of the Jetpack monorepo `docker:` commands via the Jetpack CLI

#### Jetpack product discussion

* NA

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

* In the root `package.json` you will find the `docker:` scripts. With this branch pulled down, you will see the majority of these listed when running `jetpack docker`.
* Running `jetpack docker <cmd>` should be akin to running the script directly
  (e.g. `yarn docker:jt-up` == `jetpack docker jt-up`)
* Proofread command descriptions.

### CODE REVIEW

**Comment by kraftbj** [comment]
I really like this approach. It brings the visibility of the existing commands to the CLI while giving us the future flexibility to modify them to use more direct/powerful NPM packages.

**Comment by sdixon194** [comment]
Nice - this works and it's great people able to spin up docker from anywhere. 

For some commands that take flags/arguments (`wp`, `-d`, etc), would that be handled in a separate PR?

**Comment by samiff** [comment]
> For some commands that take flags/arguments (`wp`, `-d`, etc), would that be handled in a separate PR?

@sdixon194 Great catch on the `wp` command, that functionality hadn't crossed my mind yet.

I'm going to remove the [strictCommands()](https://github.com/yargs/yargs/blob/master/docs/api.md#strictcommandsenabledtrue), and replace it with [recommendCommands()](https://github.com/yargs/yargs/blob/master/docs/api.md#recommendcommands) instead. This should give us more flexibility when building out commands, but still offer help when users enter wrong commands. cc: @kraftbj

Arbitrary input should now be passed to the various Docker commands such as:

* `jetpack docker phpunit --filter=Protect`
* `jetpack docker wp user list --role=subscriber > test.txt`

**Comment by sdixon194** [comment]
Nice! This is real slick and works for me 👍

File: tools/cli/cliRouter.js [hunk 1]
```diff
@@ -42,7 +41,7 @@ export async function cli() {
 	argv
 		.showHelpOnFail( true )
 		.demandCommand()
-		.strictCommands()
+		.recommendCommands()
```
Comment (kraftbj) [inline tools/cli/cliRouter.js:44]: What's your thought on dropping `strictCommands()`. They do work together, e.g. `jetpack dockers` will suggest `docker` while `jetpack regieergear` will exit with no output without `strictCommands`. With it, it'll tell me it is an unsupported command with the output of the help text.

**Comment by kraftbj** [comment]
`strictCommands` causes issues specifically for `jetpack docker wp` since `jetpack docker wp user list` involves yargs thinking `user, list` are unknown commands since we don't specifically define them.

If nothing else, for a `wp` passthrough, we can't account for everything and since this tool is meant to be more of a switchboard to other things than the One Thing, removing it helps overall.

Ideally, if `jetpack <cmd>` could output a default text if nothing is matched, that would be swell, but not a blocker for this.

**Review by kraftbj — APPROVED** [review]

**Pull request merged**